wp/wp-content/themes/lahma_maker/template-parts/content-arcsi.php

```diff
@@ -58,6 +84,14 @@ function sortAssociativeArrayByKey($array, $key, $direction){
 ?>
 
 <?php 
+// check if Arcsi is available
+if (!isDomainAvailible($server)) {
+    echo '<center><h3>';
+    echo 'الدمام 🤯 Arcsi is ❌ not available 🚧 at the moment 😬 نحن اسفون <br/>
+    🧤 🔥 🎯 Please try again later 👉 💫 🔊';
+    echo '</center></h3>';
+}
+
```

Comment:
Did you try this in action? We use dev arcsi to simulate a server down state.

Reply:
Typo in `isDomainAvailible` :)

Reply:
ah, ok. I'll fix it :-)

Reply:
> Did you try this in action? We use dev arcsi to simulate a server down state.

@baxgas how about this?